src/ci-line-chart.js

```diff
@@ -226,12 +228,17 @@ const vis = {
           x: {
             type: 'time',
             time: {
-              tooltipFormat: 'DD T'
+              tooltipFormat: 'DD',
+              unit: 'day',
+              round: 'day',
```

Comment:
I think we'd want to do some automated detection on whether the data are dates, timestamps etc. But for now to support opmon let's do date as default.

Reply:
Yep, I believe this is covered by #11